cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -3596,6 +3596,142 @@ action.cyences_notable_event_action.attacker_drilldown = `cs_linux_sudousers` $r
 action.cyences_send_email_action = 1
 
 
+[Linux - cs_liunx_groups_list.csv Lookup Gen]
```

Comment:
Can we use kvstore instead of csv as for users entries will be huge number as number of hosts grow?